units/commit_queue.go

```diff
@@ -66,6 +66,30 @@ func NewCommitQueueJob(env evergreen.Environment, queueID string, id string) amb
 	return job
 }
 
+func (j *commitQueueJob) TryUnstick(cq *commitqueue.CommitQueue) {
+	//unstuck the queue if the patch is done.
+	nextItem := cq.Next()
+
+	done, err := IsPatchDone(nextItem.Issue)
```

Comment:
I think for commit queues in PR mode we need to look for the item's `Version`, since the `Issue` corresponds to the PR number. The version id string will be the same as the patch's ObjectId.

Reply:
Can we just look at the Version in general then since CLI items should also have a version built by this time?

Reply:
I think we only set the version field for GitHub items. I could be wrong, though.

Reply:
Oh that's true--I thought you meant version objects vs patch objects, in which case Issue is also the version ID for CLI items.

Reply:
Changed to only dequeue if projectRef.CommitQueue.PatchType == commitqueue.CLIPatchType